src/theory/datatypes/theory_datatypes.cpp

```diff
@@ -583,6 +584,7 @@ Node TheoryDatatypes::expandDefinition(Node n)
       }
     }
   }
+  Node n_ret;
```

Comment:
```suggestion
  Node ret;
```

(naming conventions)